pkgs/top-level/all-packages.nix

```diff
@@ -7794,6 +7794,8 @@ in
 
   pycangjie = pythonPackages.pycangjie;
 
+  pycflow2dot = python.pkgs.pycflow2dot;
```

Comment:
```suggestion
  pycflow2dot =  with python3Packages; toPythonApplication pycflow2dot;
```

Reply:
this application works with python 2 and 3
and the manual prescribes the use of `pythonPackages` in such a statement
why is explicitly setting a python version preferred here?

also, does this mean the line immediately above this one is wrong, and the one 5 below?
(they're the ones i took my queue from)

(side question while i'm being nitpicky, is `pythonPackages` preferred over `python.pkgs`? which is shorter and doesn't require the shift key
and as i understand chapter 15.20.1.1.1 of the manual, the longer one is just an alias to the shorter)